.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+name: php CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        php-version: ['7.2']
```

I think this key is wrong:
```suggestion
        php_version: ['7.2']
```